R/ParamSet.R

```diff
@@ -188,6 +188,20 @@ ParamSet = R6Class("ParamSet",
       return(self)
     },
 
+    #' @description
+    #' Sets the initial parameter values of the ParamSet. They are set in the slot `values` but
+    #' are also saved and can always be accessed using the slot `initial_values`.
+    #' This function should only be called once.
```

Comment:
Otherwise I would prefer that solution: no need to have `set_initial_values` for paramsetcollection either, in that case. could even make it part of the `ps()`, `p_dbl()` etc machinery: `p_dbl(lower = 0, upper = 1, init = 0.5)`

Reply:
I prefer your solution too (as usual ;))

Reply:
But are there never situations where it is useful to set initial values for a `ParamSetCollection` that differ from the initial values of the contained `ParamSet`s?

Reply:
Afaik ParamSetCollection is usually used for composite objects (e.g. graphs of pipepos), having initial values here that differ from the initializations of the components would probably be confusing.